### PR TITLE
6 get assessment method

### LINKED ISSUE #6: Get Assessment Content from our API
Write a function to get the Assessment Content from our API - we may need to use superagent.

### PR DESCRIPTION
**Pull request by benlittlenz**
### added api folder and assessments get method

Added the api folder to store our api methods and added the get methods. 

Closes #6

### CODE REVIEW

**Comment by chrislaupama** [comment]
@nzbenji is this needed for review?

**Review by emilyparkes — COMMENTED** [review]
please remove the underscore from your file naming, the preferred option would be assessmentApi.js or if it is in a folder called api you may just name it assessments.js

File: client/api/assessments_api.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+const api = 'http://localhost:3000/api/v1'
+
+export function getAssessments () {
+  return request.get(api)
+    .then(res => {
+      res.json(res.body)
+    })
+    .catch(() => {
+      throw Error('you got an error')
```
Comment (chrislaupama) [inline client/api/assessments_api.js]: Maybe look at providing a more meaningful error message using err.message

`.catch((err) => res.json({Okay: false, error: err.message}))`

**Review by chrislaupama — CHANGES_REQUESTED** [review]
Maybe have the error pop out an error message

File: client/api/assessments.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import request from 'superagent'
+
+const URL = 'http:localhost:3000/v1/api'
```
Comment (emilyparkes) [inline client/api/assessments.js]: I believe this api should be `/api/v1/assessments` as stated in server.js

**Review by emilyparkes — DISMISSED** [review]

File: client/api/assessments.js (hunk 2 above)
Comment (benlittlenz) [reply client/api/assessments.js]: Sorry you are right. Has been changed

File: client/api/assessments.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import request from 'superagent'
+
+const URL = 'http:localhost:3000/v1'
```
Comment (emilyparkes) [inline client/api/assessments.js]: Please have this as `http:localhost:3000/api/v1/assessments` as others will need to use this as well in this file.

File: client/api/assessments.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+export const assessmentContent = (assesmentId) => {
+  return request
+    .get(`${URL}/assessments/${assesmentId}`)
```
Comment (emilyparkes) [inline client/api/assessments.js]: This will be `.get(`${URL}/`)`

**Review by emilyparkes — CHANGES_REQUESTED** [review]

**Review by emilyparkes — APPROVED** [review]

File: client/api/assessments.js (hunk 4 above)
Comment (benlittlenz) [reply client/api/assessments.js]: added assessments to URL binding

File: client/api/assessments.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import request from 'superagent'
+
+const URL = 'http:localhost:3000/v1/assessments'
```
Comment (emilyparkes) [inline client/api/assessments.js]: Have a look at what the api is actually called and make changes here

**Review by chrislaupama — DISMISSED** [review]

**Review by emilyparkes — CHANGES_REQUESTED** [review]

File: client/api/assessments.js (hunk 5 above)
Comment (benlittlenz) [reply client/api/assessments.js]: Fixed spelling error in URL and resolved conflicts

**Review by emilyparkes — APPROVED** [review]

**Pull request merged**